beets/dbcore/types.py

```diff
@@ -183,6 +183,17 @@ def format(self, value):
         return u'{0:.1f}'.format(value or 0.0)
 
 
+class PaddedFloat(Float):
```

Comment:
I think it would be fine to just extend the `Float` type rather than adding a new one. The constructor can take an _optional_ `digits` parameter that defaults to 1 (i.e., `def __init__(self, digits=1): ...`).